dagrt/builtins_python.py

```diff
@@ -59,6 +78,11 @@ def builtin_dot_product(a, b):
     return np.vdot(a, b)
 
 
+def builtin_cumulative_product(a, axis):
```

Comment:
Could you explain the use case?

Reply:
This was only being used in one place in BDF, and I managed to remove it (562b2b5)